templates/digital-asset-template/frontend/utils/truncateAddress.ts

```diff
@@ -0,0 +1,3 @@
+export function truncateAddress(address: string) {
```

Comment:
I think you can import it from the adapter package? https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-ant-design/src/utils.ts

tho we might need it also exported from the shadcn package cc @blakezimmerman

Reply:
This will be exported from `@aptos-labs/wallet-adapter-react` once we merge my PR.